### ISSUE TITLE
Locus has no credible set

### ISSUE THREAD

**Issue by AI-10** [omerwe/polyfun#177]
hi !

I'm executing polufun+susie on some regions of 500kb size, but I noticed one locus has no credible set in it? Is that normal?

**Comment by omerwe** [comment]
Yes, it's possible, because SuSiE uses a different definition of 95% credible set than previous definition. Please see [the Supplementary information of the PolyFun paper for details](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7710571/bin/NIHMS1634790-supplement-1.pdf) (specifically the lines 2-4 in the section titled "Secondary analyses of PolyFun simulation studies").
